x/auction/types/msg.go

```diff
@@ -52,20 +46,12 @@ func (msg MsgPlaceBid) ValidateBasic() error {
 
 // GetSignBytes gets the canonical byte representation of the Msg.
 func (msg MsgPlaceBid) GetSignBytes() []byte {
-	bz := ModuleCdc.MustMarshalJSON(msg)
+	bz := ModuleCdc.MustMarshalJSON(&msg)
 	return sdk.MustSortJSON(bz)
 }
 
 // GetSigners returns the addresses of signers that must sign.
 func (msg MsgPlaceBid) GetSigners() []sdk.AccAddress {
-	return []sdk.AccAddress{msg.Bidder}
-}
-
-func (msg MsgPlaceBid) String() string {
-	// String implements the Stringer interface
-	return fmt.Sprintf(`Place Bid Message:
-	Auction ID:         %d
-	Bidder: %s
-	Amount: %s
-`, msg.AuctionID, msg.Bidder, msg.Amount)
+	bidder, _ := sdk.AccAddressFromBech32(msg.Bidder)
```

Comment:
I've found ignoring error is very rarely worth it. I'd panic here.

Reply:
https://github.com/Kava-Labs/kava/pull/1052/commits/e60444b1fab6e4778cca0a2e03dd142437fb08f5